Source/JavaScriptCore/b3/B3Value.cpp

```diff
@@ -995,6 +996,10 @@ ValueKey Value::key() const
         if (numChildren() == 2)
             return ValueKey(kind(), type(), as<SIMDValue>()->simdInfo(), child(0), child(1), nullptr);
         return ValueKey(kind(), type(), as<SIMDValue>()->simdInfo(), child(0), child(1), child(2));
+    case VectorRelaxedSwizzle:
```

Comment:
Let's add this to the 2 child case above, which should handle asserting that the number of children is always 2